build/cmake/Dockerfile

```diff
@@ -13,7 +13,7 @@ RUN curl -L https://github.com/Kitware/CMake/releases/download/v3.13.4/cmake-3.1
     cd /tmp && tar xf cmake.tar.gz && cp -r cmake-3.13.4-Linux-x86_64/* /usr/local/
 
 # install boost
-RUN curl -L https://dl.bintray.com/boostorg/release/1.67.0/source/boost_1_72_0.tar.bz2 > /tmp/boost.tar.bz2 &&\
+RUN curl -L https://boostorg.jfrog.io/artifactory/main/release/1.72.0/source/boost_1_72_0.tar.bz2 > /tmp/boost.tar.bz2 &&\
```

Comment:
I'm actually not sure if the old download link was for 1.67 or 1.72 here... based on the hash, it looks like this was downloading 1.67.

Reply:
I agree - if you're going to bump to 1.72 you need to change the hash as well.

Reply:
https://github.com/apple/foundationdb/pull/4771#discussion_r626873763 seems like this file is more or less unused, so I'll leave the download as 1.67 here. We can always update it in the future if necessary.